<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-CQO0GVPR" failures="2" tests="7" name="BankingApp Test" time="90.004" errors="0" timestamp="2023-06-04T18:48:57 AEST">
  <testcase classname="com.inetbanking.testCases.TC_LoginTest_001" name="logintest" time="7.528"/>
  <testcase classname="com.inetbanking.testCases.TC_LoginTestDDT_002" name="logintest" time="11.273">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTestDDT_002.logintest(TC_LoginTestDDT_002.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase classname="com.inetbanking.testCases.TC_LoginTestDDT_002" name="logintest" time="5.416"/>
  <testcase classname="com.inetbanking.testCases.TC_LoginTestDDT_002" name="logintest" time="7.527">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTestDDT_002.logintest(TC_LoginTestDDT_002.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase classname="com.inetbanking.testCases.TC_CreateNewCustomer_001" name="createNewCustomerTest" time="15.321"/>
  <testcase classname="com.inetbanking.testCases.TC_CreateNewCustomer_002" name="createNewCustomerTest" time="13.994"/>
  <testcase classname="com.inetbanking.testCases.TC_CreateNewCustomer_003" name="resetNewCustomerDetailsTest" time="15.194"/>
</testsuite> <!-- BankingApp Test -->
